great_expectations/data_context/config_validator/yaml_config_validator.py

```diff
@@ -153,7 +153,9 @@ def test_yaml_config(  # noqa: C901 - complexity 17
 
         Args:
             yaml_config: A string containing the yaml config to be tested
-            name: (Optional) A string containing the name of the component to instantiate
+            name: Optional name of the component to instantiate
+            class_name: Optional, overridden if provided in the config
+            runtime_environment: Optional override for config items
```

Comment:
Fixed the issues caught by the docstring linter by adding these two parameters to the args list.

Reply:
Actually nevermind, the actual issues are in abstract_data_context.py, I just replicated those changes here.